### ISSUE TITLE
there is no clear documentation of this package

### ISSUE THREAD

**Issue by ayuningnareswari** [tumtumtum/Shaolinq#89]
Please write full documentation including Anotation, Pivot table (if exist), check table exist, attach / detach, etc. Just like documentain on Laravel ORM.

**Comment by tumtumtum** [comment]
Agreed. This is a open source project and we're always looking for volunteers to help...especially with documentation.
